src/views/HomePage.vue

```diff
@@ -6,15 +6,25 @@
       <TopNavBar />
       <div v-if="!data?.recommendations">로딩중입니다.</div>
       <Carousel :recommendations="data?.recommendations" v-else />
+      <NextButton :onClick="moveToSizePage">꿀조합 찾기</NextButton>
+      <GlobalNavBar />
     </section>
   </div>
 </template>
 
 <script setup lang="ts">
+import { useRouter } from 'vue-router';
+
 import TopNavBar from '@containers/TopNavBar.vue';
 import Carousel from '@containers/Carousel.vue';
-
+import NextButton from '@components/NextButton.vue';
+import GlobalNavBar from '@containers/GlobalNavBar.vue';
 import { useGetRecommendations } from '@apis/recommendations';
 
 const { data } = useGetRecommendations();
+const router = useRouter();
+
+function moveToSizePage() {
+  router.push('/size');
+}
```

Comment:
되도록 화살표 함수로 통일해 사용해주세요.
그리고 이런 메서드를 사용할 경우에는 굳이 함수화 할 필요없이 template에 직접 적용하는 것이 더 깔끔합니다.
onClick의 경우 리액트의 이벤트 이름과 동일하기 때문에 헷갈릴 수 있습니다.
최대한 제공되는 이벤트명을 피해서 지어주시면 더 좋을 것 같습니다!

```suggestion
<NextButton :onClick=" router.push('/size')">꿀조합 찾기</NextButton>
```

Reply:
NextButton 클릭 시 여러 함수를 실행할 수 있을 것 같아서 조금 큰 범위로 작명을 해봤는데, 좀더 명확한 네이밍을 찾아봐야겠네요 :)!!!